packages/trader/src/Modules/Trading/Helpers/contract-type.js

```diff
@@ -10,6 +10,12 @@ export const contract_category_icon = {
     [localize('Multipliers')]: 'IcMultiplier',
 };
 
+export const contract_type_category_icon = {
```

Comment:
This needs to be a function for `localize` to work.

Reply:
@zakhov just handled this.